### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Saqibi4213**
i enjoy creating tribute page
My project is about Ahmad shah baba i made tribute page about it 
i add Html file 
added CSS file 
i just design my project trough 2 languages HTML , CSS  
and must say first i try to do in FREECODECAMP then 
make repo for my project 
git add .
git commit 
git push
 please Sir  review my project and give me feedback 
THANK YOU!

### CODE REVIEW

File: index.html
Comment (ahzamir) [inline index.html:1]: - Your HTML code spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  <body>
+    <main id="main">
+    <h1 id="title">Ahmad Shah Baba</h1>
+   <p>
+ <div id="img-div">
+    <img 
+    id="image" 
+    src="C:\Users\Power link\Downloads\images.jpg" 
+    alt="Ahmad Shah Baba" 
+    class="centered-image"align="center";>
```
Comment (ahzamir) [inline index.html]: - Your image is not showing in the browser and the problem is that you linked the image from your local machine which is wrong:
  - You have to add an external (online) link for your image.
  - Or if you want to add an image from your local machine then you have to add the image to your project folder and then add the link of the image folder in your img tag.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Saqibi4213 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although your code passes the freecodecamp website, but you have to make some changes to your code: **Check the inline comments below**

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

SoCloseKeeganMichaelKeyGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Tribute Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your Code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Review by ahzamir — CHANGES_REQUESTED** [review]
@Saqibi4213 

- Still, your code is not clean:
- And for cleaning your code:
  - At the bottom right side of your vscode click on `Spaces` then click on `Index using spaces` and choose 2 from there.
  - Then according to the lines on the screen all child elements should be 2 spaces or one line ahead of their parent element and the closing tag should be in the same line as the opening tag!
  
  - Your body spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

**Comment by Saqibi4213** [comment]
hello sir @ahzamir  
i try to clear my code 
please re-review my project and let me know if thier is any mistakes left from me and if thier is no mistake all is clear and good again give me feedback 
thank you !

**Review by ahzamir — CHANGES_REQUESTED** [review]
@Saqibi4213 

- Although you cleaned some of your code and aligned the opening tags but you did not clean and aligned the closing tags.
- Your HTML code spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.
  - **The Closing tags should be inline with the opening tag or in the same line of spacing as its opening tag**

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

**Comment by Saqibi4213** [comment]
Hello Sir @ahzamir  
I try my best to catch all errors and problems my code have and try to clear,clean my code 
please Re-review if thier is still issue or not 
Regards!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Ahmad Shah Baba</h1>
+      <div id="img-div">
+        <img id="image" src="C:\Users\Power link\Downloads\images.jpg" alt="Ahmad Shah Baba" class="centered-image" align="center">
```
Comment (ahzamir) [inline index.html:12]: - [SUGESSION] - As I can see your image is not showing up in the browser and the problem is that you added the image from your computer and provided the folder link from your computer which is wrong.
  - To solve this, add the image to your project main directory and then use the image folder link in href.

**Review by ahzamir — APPROVED** [review]
### Hi @Saqibi4213 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Tribute Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**